libs/components/i18n/schematics/ng-add/ng-add.schematic.spec.ts

```diff
@@ -19,7 +19,7 @@ describe('ng-add.schematic', () => {
     runner = new SchematicTestRunner('schematics', COLLECTION_PATH);
 
     tree = await createTestLibrary(runner, {
-      name: defaultProjectName,
+      projectName: defaultProjectName,
```

Comment:
I updated the `testing/scaffold.ts` file to match what we're using in `@skyux/packages` which led to a handful of inconsequential changes in the tests.